lib/datasets/wine.rb

```diff
@@ -0,0 +1,64 @@
+require 'csv'
+
+require_relative 'dataset'
+
+module Datasets
+  class Wine < Dataset
+    Record = Struct.new(:wine_class,
+                        :alcohol,
+                        :malic_acid,
+                        :ash,
+                        :alcalinity_of_ash,
+                        :magnesium,
+                        :total_phenols,
+                        :flavanoids,
+                        :nonflavanoid_phenols,
+                        :proanthocyanins,
+                        :color_intensity,
+                        :hue,
+                        :od280_od315_of_diluted_wines,
```

Comment:
I don't know much about this area but `optical_density` may be better.

Reply:
I searched, use `optical_nucleic_acid_concentration`. ref: https://www.gelifesciences.co.jp/newsletter/biodirect_mail/chem_story/138.html (in Japanese)

Reply:
OK!